### PR TITLE
Split basic configuration

### PR DESCRIPTION
**Pull request by goreck888**

### CODE REVIEW

**Comment by agpul** [comment]
Can you provide me a short scenario of how this PR can be tested?

**Review by agpul — APPROVED** [review]

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 class PagesController < ApplicationController
   def about
+    ab_finished(:recommendations)
```
Comment (JanKapala) [inline app/controllers/pages_controller.rb:5]: There is no ab_test funstion usage. Maybe should it be added?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -47,4 +47,8 @@ def meta_robots
   def yield_content!(content_key)
     view_flow.content.delete(content_key)
   end
+
+  def placeholder(variant, text = "Placeholder - link to about", link = about_path)
+    render "layouts/placeholder", text: text, link: link, variant: variant
+  end
```
Comment (JanKapala) [inline app/helpers/application_helper.rb:53]: There is no usage of this placeholder. did you mean to use it with ab_test?

File: config/experiments.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+recommendations:
+  alternatives:
+    - name: top
+      percent: 25
+    - name: middle
+      percent: 25
+    - name: none
+      percent: 50
```
Comment (JanKapala) [inline config/experiments.yml:8]: This experiment isn't displayed in the Split Dashboard

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (JanKapala) [inline app/helpers/application_helper.rb:53]: There is no usage of this placeholder. Did you mean to use it with ab_test?

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (JanKapala) [inline app/controllers/pages_controller.rb:5]: There is no ab_test function usage. Maybe should it be added?

**Review by JanKapala — CHANGES_REQUESTED** [review]
The ab_finished function is used but ab_test isn't so an experiment is not displayed in the Split Dashboard. Maybe should it be added to make example of usage A/B testing more end-to-end? @goreck888

File: config/experiments.yml (hunk 3 above)
Comment (goreck888) [reply config/experiments.yml:8]: Yes, I removed experiment used for tests in the last commit, just forget about remove it in config files.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (goreck888) [reply app/helpers/application_helper.rb:53]: I wanted to keep it for future implementations of tests but I can remove it.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (goreck888) [reply app/controllers/pages_controller.rb:5]: I forgot about it

**Review by wujuu — APPROVED** [review]

**Review by JanKapala — APPROVED** [review]

**Pull request merged**